semanticdb3/semanticdb3.md

```diff
@@ -0,0 +1,491 @@
+# SemanticDB Specification, Version 3.0.0
+
+  * [Motivation](#motivation)
+  * [Overview](#overview)
+  * [Model](#model)
+    * [TextDocument](#textdocument)
+    * [Range](#range)
+    * [Symbol](#symbol)
+    * [Identifier](#identifier)
+    * [SymbolInformation](#symbolinformation)
+    * [Diagnostic](#diagnostic)
+    * [Synthetic](#synthetic)
+  * [Schemas](#schemas)
+    * [Protobuf](#protobuf)
+    * [JSON](#json)
+    * [SQL](#sql)
+  * [Changelog](#changelog)
+    * [3.0.0](#300)
+
+## Motivation
+
+Nowadays, there is a clear trend towards standards for communication between developer tools.
+Language Server Protocol (LSP) [\[2\]][2], a protocol that connects programming
+language implementations and text editors, has gained strong industrial support
+and at the time of writing has implementations for over 25 programming languages.
```

Comment:
and most major editors. I don't think there is any editor with >2% marketshare that doesn't have some LSP client implementation, maybe nano but that still doesn't disqualify "most major editors"

Reply:
I'll think about diplomatic formulation that does LSP justice.